Correct config/example.json

Since externs/vbarray.js has been deleted on 3.5.0, we should remove it from the config/example.json 

See https://github.com/openlayers/ol3/blob/master/config/example.json#L24

Indeed.

```
$ make build/compiled-examples/simple.combined.js
node tasks/build.js build/compiled-examples/simple.json build/compiled-examples/simple.combined.js
info ol Parsing dependencies
info ol Compiling 352 sources
ERR! compile ERROR - Cannot read: externs/vbarray.js
ERR! compile 
ERR! compile 
ERR! compile 1 error(s), 0 warning(s)
ERR! compile 
ERR! Process exited with non-zero status, see log for more detail: 1 
make: *** [build/compiled-examples/simple.combined.js] Error 1
rm build/compiled-examples/simple.json
```

I'll provide a PR.

--- a/config/example.json
+++ b/config/example.json
@@ -20,8 +20,7 @@
       "externs/jquery-1.9.js",
       "externs/proj4js.js",
       "externs/tilejson.js",
-      "externs/topojson.js",
-      "externs/vbarray.js"
+      "externs/topojson.js"
     ],
     "define": [
       "goog.array.ASSUME_NATIVE_FUNCTIONS=true",

